recipes/pyremap/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyremap" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xylar/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3ca8da6fdefaf23f14a4556271e025aef0ab5480a0cae5da1cc3fa1bb46f82f3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

Comment:
Can this be noarch? If not the python version should be constrained with skip instead

Reply:
Sorry about that.  I had it as noarch before and realized it doesn't work under Windows.  I have no desire to make it Windows compatible.  I'll change the recipe to skip `py<36` as well and take out the python constraint.